### ISSUE TITLE
MSSQL DELETE_ALL tests fail on appveyor see PR #72 

### ISSUE THREAD

**Issue by photodude** [joomla-framework/database#74]
### Steps to reproduce the issue
MSSQL DELETE_ALL tests fail on appveyor see PR #72 

### Expected result
tests pass

### Actual result
tests fail 
```bat
There were 36 errors:
1) Joomla\Database\Tests\DriverSqlsrvTest::test__destruct
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
2) Joomla\Database\Tests\DriverSqlsrvTest::testConnected
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
3) Joomla\Database\Tests\DriverSqlsrvTest::testDropTable
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
4) Joomla\Database\Tests\DriverSqlsrvTest::testEscape with data set #0 (''%_abc123', false, '''%_abc123')
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
5) Joomla\Database\Tests\DriverSqlsrvTest::testEscape with data set #1 (''%_abc123', true, '''%[_]abc123')
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
6) Joomla\Database\Tests\DriverSqlsrvTest::testGetAffectedRows
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
7) Joomla\Database\Tests\DriverSqlsrvTest::testGetCollation
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
8) Joomla\Database\Tests\DriverSqlsrvTest::testGetExporter
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
9) Joomla\Database\Tests\DriverSqlsrvTest::testGetImporter
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
10) Joomla\Database\Tests\DriverSqlsrvTest::testGetNumRows
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
11) Joomla\Database\Tests\DriverSqlsrvTest::testGetTableCreate
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
12) Joomla\Database\Tests\DriverSqlsrvTest::testGetTableColumns
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
13) Joomla\Database\Tests\DriverSqlsrvTest::testGetTableKeys
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
14) Joomla\Database\Tests\DriverSqlsrvTest::testGetTableList
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
15) Joomla\Database\Tests\DriverSqlsrvTest::testGetVersion
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
16) Joomla\Database\Tests\DriverSqlsrvTest::testInsertid
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
17) Joomla\Database\Tests\DriverSqlsrvTest::testLoadAssoc
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
18) Joomla\Database\Tests\DriverSqlsrvTest::testLoadAssocList
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
19) Joomla\Database\Tests\DriverSqlsrvTest::testLoadColumn
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
20) Joomla\Database\Tests\DriverSqlsrvTest::testLoadObject
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
21) Joomla\Database\Tests\DriverSqlsrvTest::testLoadObjectList
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
22) Joomla\Database\Tests\DriverSqlsrvTest::testLoadResult
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
23) Joomla\Database\Tests\DriverSqlsrvTest::testLoadRow
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
24) Joomla\Database\Tests\DriverSqlsrvTest::testLoadRowList
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
25) Joomla\Database\Tests\DriverSqlsrvTest::testExecute
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
26) Joomla\Database\Tests\DriverSqlsrvTest::testRenameTable
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
27) Joomla\Database\Tests\DriverSqlsrvTest::testSelect
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
28) Joomla\Database\Tests\DriverSqlsrvTest::testSetUtf
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
29) Joomla\Database\Tests\DriverSqlsrvTest::testIsSupported
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
30) Joomla\Database\Tests\DriverSqlsrvTest::testUpdateObject
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
31) Joomla\Database\Tests\IteratorSqlsrvTest::testForEach with data set #0 ('title', '#__dbtest', null, 'stdClass', 0, 0, array(stdClass Object (...), stdClass Object (...), stdClass Object (...), stdClass Object (...)), null)
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
32) Joomla\Database\Tests\IteratorSqlsrvTest::testForEach with data set #1 ('title', '#__dbtest', null, 'stdClass', 2, 0, array(stdClass Object (...), stdClass Object (...)), null)
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
33) Joomla\Database\Tests\IteratorSqlsrvTest::testForEach with data set #2 ('title', '#__dbtest', null, 'stdClass', 20, 2, array(stdClass Object (...), stdClass Object (...)), null)
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
34) Joomla\Database\Tests\IteratorSqlsrvTest::testForEach with data set #3 ('title, id', '#__dbtest', 'title', 'stdClass', 0, 0, array(stdClass Object (...), stdClass Object (...), stdClass Object (...), stdClass Object (...)), null)
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
35) Joomla\Database\Tests\IteratorSqlsrvTest::testForEach with data set #4 ('title', '#__dbtest', 'title', 'UnexistingClass', 0, 0, array(), 'InvalidArgumentException')
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
36) Joomla\Database\Tests\IteratorSqlsrvTest::testCount
PHPUnit_Extensions_Database_Operation_Exception: COMPOSITE[DELETE_ALL] operation failed on query: 
                DELETE FROM jos_dbtest
             using args: Array
(
)
 [SQLSTATE[42S02]: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Invalid object name 'jos_dbtest'.]
C:\projects\database\vendor\joomla\test\src\TestDatabase.php:218
```

### Additional comments
Possible reference items for solving the issues
- http://stackoverflow.com/questions/30939118/error-when-using-phpunit-dbunit-with-sql-server
- http://stackoverflow.com/questions/10948273/loading-schema-from-sql-file-in-phpunit-test

**Comment by photodude** [comment]
Looks like these issues are specific to the joomla/test package as noted in the errors.
see the Joomla/test::setUp() for the root https://github.com/joomla-framework/test/blob/master/src/TestDatabase.php#L202-L219

Might be related to the issue listed in this blog
https://andreas.heigl.org/2015/08/16/database-testing-with-phpunit-and-sqlite/

**Comment by photodude** [comment]
Closing as the fix in in PR #72 
see https://andreas.heigl.org/2015/08/16/database-testing-with-phpunit-and-sqlite/ for solution
